std/typecons.d

```diff
@@ -2801,9 +2784,18 @@ succeeds, `this` becomes non-null.
 Params:
     value = A value of type `T` to assign to this `Nullable`.
  */
-    void opAssign()(T value)
+    @trusted void opAssign()(T value)
```

Comment:
How can we be sure that this is always `@safe`?
Wouldn't it be better to make move/moveEmplace due the inference (and rather fix that than blasting things with @trusted here)?

Reply:
Because `move` is @safe and `moveEmplace` is only unsafe because it calls memcpy, which we know to be safe because the target variable is only initialized to `.init` and no methods have been called on it yet, or `.destroy` has been called on it; semantically it's uninitialized memory.